ArduPlane/mode.h

```diff
@@ -225,6 +225,9 @@ class ModeGuided : public Mode
     // handle a guided target request from GCS
     bool handle_guided_request(Location target_loc) override;
 
+    // adjustable radius for guided. A radius of Zero indicates to use WP_LOITER_RAD
+    uint16_t active_radius_m;
```

Comment:
fine with me. Only reason I use uint16 is because it retains the knowledge that the direction is not known here and that the L1 controller only accepts unsigned. From the "app" level up in the plane, I agree that we should have as much info as possible, for the example case of replacing L1 with something in the future.

Reply:
changed to float